tests/test_symlink.py

```diff
@@ -0,0 +1,19 @@
+import os
+import shutil
+
+import audeer
+
+
+def test_symlinks(tmpdir):
+    program = 'ffmpeg'
+    path = audeer.mkdir(os.path.join(tmpdir, 'bin'))
+    command = shutil.which(program)
+    assert os.path.exists(command)
+    local_command = os.path.join(path, program)
+    os.symlink(command, local_command)
+    # shutil.copyfile(command, local_command)
+    assert os.path.exists(local_command)
+    print(os.environ['PATH'])
+    os.environ['PATH'] = path
+    print(os.environ['PATH'])
+    assert shutil.which(program) == local_command
```

Comment:
Maybe the copying solution does not work due to missing dependencies? We probably have to copy some shared libraries as well.

Reply:
Maybe the copying solution does not work due to missing dependencies? We probably have to copy some shared libraries as well.

Reply:
Locally it works for me when doing it with Unix commands.
In the Github runner it does not fail when executing `ffmpeg`, but it cannot find it in the path: https://github.com/audeering/audiofile/runs/5756308827?check_suite_focus=true

Reply:
Did you try with `.exe`?

Reply:
According to https://stackoverflow.com/questions/56399696/shutil-which-not-finding-programs-without-appending-file-extension: "shutil.which('ffmpeg') will look for "ffmpeg.EXE" in PATH", but I guess the executable is called `ffmpeg.exe`.

Reply:
Renaming it to `ffmpeg.EXE` did indeed help and the custom `tests/test_symlink.py` no works.

With the actual audio tests we still get 4 failures under Windows for which I'm not sure why yet: https://github.com/audeering/audiofile/runs/5770153335?check_suite_focus=true

Reply:
I guess best would be to run it on a Windows machine locally and debug what's going on. I can later give it a try on my wife's machine if you want.

Reply:
That would be nice.
If you don't need this fix immediately it can also wait for a couple of days.

Reply:
At least locally tests are passing on Windows:

![image](https://user-images.githubusercontent.com/10383417/161398134-43f18ecf-2007-4e17-830e-610668b850a7.png)

So maybe we simply skip the failing tests on Windows?

Reply:
I'm not a big fan of skipping the tests. But as we seem to need a fix as soon as possible and we don't have time to first investigate https://github.com/audeering/audiofile/issues/65 it might be ok for an intermediate release to skip the test.

I will take a look and update the code accordingly.

Reply:
On the other hand this problem would disappear when we just remove `sox`.

I started running the benchmarks with and without `sox`, let's first see what the outcome will be.

Reply:
I solved it in https://github.com/audeering/audiofile/pull/69 by using Github runners with and without `sox` installed. Then we don't need to mess around with the path as we do here.

Reply:
Hehe, I also thought about using different Github runners, but didn't want to suggest it after you already spend to effort on hiding `sox` from PATH :)